app/views/groups/_group.html.erb

```diff
@@ -0,0 +1,10 @@
+<div id="<%= dom_id group %>">
+  <div class="w-100 mt-2 p-2">
+    <div class="d-flex group p-4 justify-content-between">
+      <img src="<%= group.icon %>" style="width: 30px" />
+      <p class="my-auto"><%= group.name %></p>
+      <% sum = group.entities.sum(&:amount) %>
+       <p class="my-auto">  <%= sum %> </p>
+    </div>
+  </div>
+</div>
```

Comment:
- Kindly improve the design, Check these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
* Colors
* Typography: font face, size, and weight.
* Layout: composition and space between elements.
- [ ] Kindly follow the provided design (width, font, size, height, background, color...)
 -  Add this white border around the ADD CATEGORY button text. 👍🏻 
 - The NEW TRANSACTION button has a similar problem. Kindly fix it as well. 💯  👍🏻
 - The total amount number should be bold and the text should be normal, Please increase this font width. 👍🏻
 - The card should have a category image, name, `created date`, and amount. kindly let's display the creation date for each category card. 🛀🏻 

Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/234994302-d9b0bff7-fb2f-4229-8f1d-25922cd04eef.png) | ![image](https://user-images.githubusercontent.com/79058364/219786288-a43c8eb4-d309-4b09-9b4e-02b929ccd7fb.png)